tuf/api/metadata.py

```diff
@@ -294,9 +294,8 @@ def from_dict(cls, signed_dict: JsonDict) -> 'Signed':
         # Convert 'expires' TUF metadata string to a datetime object, which is
         # what the constructor expects and what we store. The inverse operation
         # is implemented in 'to_dict'.
-        signed_dict['expires'] = datetime.strptime(
-                signed_dict['expires'],
-                "%Y-%m-%dT%H:%M:%SZ").replace(tzinfo=None)
```

Comment:
note that I'm not doing the replace() in my version: I could not think of a case where it would be useful.